x-pack/plugins/alerting/common/alert_instance.ts

```diff
@@ -8,12 +8,12 @@
 import * as t from 'io-ts';
 import { DateFromString } from './date_from_string';
 
-const actionSchema = t.partial({
+const actionSchema = t.type({
```

Comment:
**optional nit:**

```suggestion
const throttledActionSchema = t.type({
```

Reply:
Done in [#3d137a1](https://github.com/elastic/kibana/pull/147360/commits/3d137a13893d66c7b29ac93e20446f4fefbba12c)